cirq/google/op_deserializer.py

```diff
@@ -115,12 +115,12 @@ def _args_from_proto(self, proto: v2.program_pb2.Operation
                 which = arg_proto.WhichOneof('arg')
                 if which == 'arg_value':
                     arg_value = arg_proto.arg_value
-                    which = arg_value.WhichOneof('arg_value')
-                    if which == 'float_value':
+                    which_val = arg_value.WhichOneof('arg_value')
```

Comment:
Oh man... nice defensive fix

Reply:
I can't take credit; mypy caught the issue because the WhichOneof return values are typed as unions of literals, so it could tell that `which` was getting reassigned to a different type :-)